### ISSUE TITLE
Achievement lockup

### ISSUE THREAD

**Issue by Zolisoru** [ValveSoftware/portal2#210]
Every time I've get a new achievement, the game locks up in black screen, with the achievement in the bottom right corner; to exit, I have to kill the portal2.sh and the portal2_linux processes, in this particular order.
System configuration:
Ubuntu 12.04 64bit
Kernel 3.8.0-89 generic
APU: AMD A10-7850K
RAM: 8GB(7 System + 1 Video)
OpenGL Version: 4.3.12682 Compatibility Profile Context 13.30

**Comment by Tele42** [comment]
If possible, can you test if this issue is present when using a semi-recent open source driver or catalyst 14.4?

**Comment by Zolisoru** [comment]
With Catalyst 14.4 beta is worse: it locks up the whole computer, I cannot switch workplaces & kill the processes. The behaviour it has changed a little bit: the achievement pops up, disappears, then the computer locks up. If any chance to run debug mode, I'm open to it.
System information(from Steam):

Processor Information:
    Vendor:  AuthenticAMD
    CPU Family:  0x15
    CPU Model:  0x30
    CPU Stepping:  0x1
    CPU Type:  0x0
    Speed:  3700 Mhz
    4 logical processors
    4 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Supported
    SSE41:  Supported
    SSE42:  Supported

Network Information:
    Network Speed:  

Operating System Version:
    Ubuntu 12.04.4 LTS (64 bit)
    Kernel Name:  Linux
    Kernel Version:  3.8.0-39-generic
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11303000
    X Window Manager:  Compiz
    Steam Runtime Version:  steam-runtime-release_2014-04-15

Video Card:
    Driver:  ATI Technologies Inc. AMD Radeon(TM) R7 Graphics

```
Driver Version:  4.4.12874 Compatibility Profile Context 13.30
OpenGL Version: 4.4
Desktop Color Depth: 24 bits per pixel
Monitor Refresh Rate: 59 Hz
VendorID:  0x1002
DeviceID:  0x130f
Number of Monitors:  1
Number of Logical Video Cards:  1
Primary Display Resolution:  1920 x 1200
Desktop Resolution: 1920 x 1200
Primary Display Size: 20.39" x 12.76"  (24.02" diag)
                                        51.8cm x 32.4cm  (61.0cm diag)
Primary VRAM Not Detected
```

Sound card:
    Audio device: ATI R6xx HDMI

Memory:
    RAM:  6904 Mb

Miscellaneous:
    UI Language:  English
    LANG:  en_CA.UTF-8
    Microphone:  Not set
    Total Hard Disk Space Available:  112983 Mb
    Largest Free Hard Disk Block:  79720 Mb

Installed software:

Recent Failure Reports:

**Comment by davidw-valve** [comment]
Does this issue occur if you are running the game in a window?

If it is capable of locking up the entire computer it is probably an issue with the drivers so I would suggest sending a bug report to the driver vendor and talking to them.

**Comment by Zolisoru** [comment]
Yes, it crashes windowed too, with a difference: it reboots by himself, don't have to cycle the power .
And I've reported it to AMD, too.
And I still has 33 more achievements to receive, so I don't run out of test material to soon.

**Comment by waitwhobrokethat** [comment]
Are you running Ubuntu 12.04.3?

**Comment by Zolisoru** [comment]
From the Steam system info:
Operating System Version:
Ubuntu 12.04.4 LTS (64 bit)
